README.md

```diff
@@ -9,8 +9,9 @@
 A Unity asset extractor for Python based on [AssetStudio](https://github.com/Perfare/AssetStudio).
 
 Next to extraction, UnityPy also supports editing Unity assets.
-Via the typetree structure all objects types can be edited.
-```py
+Via the typetree structure all object types can be edited.
```

Comment:
It might make sense to say ``can be edited in their native forms`` and then proceeding to say that some have handled export functions to make the export easier.